owncloudApp/src/main/java/com/owncloud/android/ui/activity/FolderPickerActivity.kt

```diff
@@ -48,19 +49,22 @@ open class FolderPickerActivity : FileActivity(),
     MainFileListFragment.FileActions {
 
     protected val mainFileListFragment: MainFileListFragment?
-        get() = supportFragmentManager.findFragmentByTag(TAG_LIST_OF_FOLDERS) as MainFileListFragment?
+       get() = supportFragmentManager.findFragmentByTag(TAG_LIST_OF_FOLDERS) as MainFileListFragment?
 
     private lateinit var pickerMode: PickerMode
 
+    private  lateinit var binding : FilesFolderPickerBinding
     override fun onCreate(savedInstanceState: Bundle?) {
         Timber.d("onCreate() start")
 
         super.onCreate(savedInstanceState)
 
+        binding = FilesFolderPickerBinding.inflate(layoutInflater)
         setContentView(R.layout.files_folder_picker)
 
         // Allow or disallow touches with other visible windows
-        val filesFolderPickerLayout = findViewById<LinearLayout>(R.id.filesFolderPickerLayout)
+
+        val filesFolderPickerLayout = binding.filesFolderPickerLayout
         filesFolderPickerLayout.filterTouchesWhenObscured = PreferenceUtils.shouldDisallowTouchesWithOtherVisibleWindows(this)
```

Comment:
This variable is not used anywhere else, so I think we could remove it, what do you think?

```suggestion
        binding.filesFolderPickerLayout.filterTouchesWhenObscured = PreferenceUtils.shouldDisallowTouchesWithOtherVisibleWindows(this)
```

Reply:
yes , we can  directly use without creating the variable ;;
Updated it